test/performance/alphabet/alphabet_to_char_benchmark.cpp

```diff
@@ -52,4 +59,29 @@ BENCHMARK_TEMPLATE(to_char_, seqan3::masked<seqan3::dna4>);
 BENCHMARK_TEMPLATE(to_char_, seqan3::qualified<seqan3::dna4, seqan3::phred42>);
 BENCHMARK_TEMPLATE(to_char_, seqan3::qualified<seqan3::dna5, seqan3::phred63>);
 
+#if SEQAN3_HAS_SEQAN2
+template <typename alphabet_t>
+void to_char_2(benchmark::State & state)
+{
+    std::array<alphabet_t, 256> alphs{};
+    size_t i = 0;
+    for (alphabet_t & a : alphs)
+        a = (char)i++;
```

Comment:
You can save the cast, if you choose `char` type for `i`:
```suggestion
    char i = 0;
    for (alphabet_t & a : alphs)
        a = i++;
```